.github/BOTMETA.yml

```diff
@@ -214,59 +216,64 @@ files:
     labels: docker_compose
   $modules/cloud/docker/docker_config.py:
     authors: chouseknecht ushuz
+    maintainers: $team_docker
```

Comment:
A few lines above we have:
```yaml
  $modules/cloud/docker/:
    authors: chouseknecht
    maintainers: $team_docker
```

So you don't need to redefine `maintainers` for individual plugins unless you want to add extra people (which we  aren't)

Reply:
done, thanks